addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/test/EmulatedV6Bridge.java

```diff
@@ -325,7 +325,7 @@ protected void logUnknownPacket(byte[] data, int len, String reason) {
         for (int i = 0; i < len; ++i) {
             s.append(String.format("%02X ", data[i]));
         }
-        logger.error(reason + ": " + s.toString());
+        logger.error("{}: {}", reason, s);
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace